akka-docs/src/main/paradox/persistence.md

```diff
@@ -68,7 +68,10 @@ development of event sourced applications (see section [Event sourcing](#event-s
 <a id="event-sourcing"></a>
 ## Event sourcing
 
-The basic idea behind [Event Sourcing](https://msdn.microsoft.com/en-us/library/jj591559.aspx) is quite simple. A persistent actor receives a (non-persistent) command
+See [here](https://msdn.microsoft.com/en-us/library/jj591559.aspx) for an introduction to EventSourcing, what follows is
```

Comment:
I would suggest avoiding the trap of "see here" links, they're not so well discoverable rather than "See an introduction to event sourcing on microsoft ..." but I can live with it I guess..